packages/zoltan2/src/algorithms/connectivity/Zoltan2_VtxLabel.hpp

```diff
@@ -398,7 +436,7 @@ class iceSheetPropagation {
         lno_t out_degree = g->out_degree(curr_node.id);
         lno_t* outs = g->out_vertices(curr_node.id);
         for(int i = 0; i < out_degree; i++){
-	  IcePropVtxLabel neighbor = femvData[outs[i]];
+	  IcePropVtxLabel<gno_t,lno_t> neighbor = femvData[outs[i]];
```

Comment:
Maybe define a type so that you don't have to type the template arguments everywhere.

Reply:
Will do.